src/GeoPing.Api/Services/Helper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GeoPing.Api.Interfaces;
+using GeoPing.Api.Models;
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api.Services
+{
+    public class Helper : IHelper
```

Comment:
Very bade name for the class, Helper is like 'Service' or 'Class' or 'Component' or 'Controller', what of the scope of it's help?

Reply:
done